docs/templates/PACKAGE_MAIN_README.MD

```diff
@@ -0,0 +1,99 @@
+<!-- TOKENS:
+- PACKAGE_NAME
+- SLOGAN
+- PACKAGE_COVER_IMAGE
+-->
+
+# [PACKAGE_NAME]
+
+<!-- HEADLINE:
+* required
+
+The package name of the library. e.g. `my-package-name`  
+If it is a scoped lib also include the scope e.g. `@scope/my-package-name`
+-->
+
+Example:
+> # @rx-angular/cdk
+
+<!-- CORE_SHIELDS:
+* required
+
+The core shields of the package:
+- Active NPM version e.g. `[![npm](https://img.shields.io/npm/v/[PACKAGE_NAME].svg)](https://www.npmjs.com/package/[my-package])`
+- CI Status e.g. [GitHub workflow status badge](https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge)
+- Coverage Status e.g. [coveralls.io](https://coveralls.io/)
+-->
+
+Example:
+> [![npm](https://img.shields.io/npm/v/%40rx-angular%2Fcdk.svg)](https://www.npmjs.com/package/%40rx-angular%2Fcdk)
+![rx-angular CI](https://github.com/rx-angular/rx-angular/workflows/rx-angular%20CI/badge.svg?branch=master)
+[![Coverage Status](https://raw.githubusercontent.com/rx-angular/rx-angular/github-pages/docs/test-coverage/cdk/jest-coverage-badge.svg)](https://rx-angular.github.io/rx-angular/test-coverage/cdk/lcov-report/index.html)
+
+
+## [SLOGAN]
+
+<!-- HEADLINE:
+* required
+
+A short sentence pitching the idea and benefits
+-->
+
+Example:
+> ## A Component Development Kit for high performant and ergonomic Angular applications and UI libs.
+
+
+<!-- TEASER:
+* required
+
+A short test summarizing goals and motivations
+-->
```

Comment:
I don't know if we just repeat ourselves here. This sounds like the Motivation section of the other MD file

Reply:
Imo, either slogan or teaser should go.